Earthfile

```diff
@@ -66,7 +66,7 @@ test:
     WORKDIR /build
     COPY +luet/luet /usr/bin/luet
     COPY . .
-    RUN go run github.com/onsi/ginkgo/v2/ginkgo --fail-fast --slow-spec-threshold 30s --covermode=atomic --coverprofile=coverage.out -p -r ./pkg ./internal ./cmd ./sdk
```

Comment:
Sticking to the default value? What would that be? Any reason we don't want to be explicit here?

Reply:
The flag has been deprecated since ginkgo 2.5, so it's not doing anything anymore and won't be valid soon